From deec4709404c0c26f8fe57bcd8bf5c04f7b6ed0b Mon Sep 17 00:00:00 2001
From: Edward Hervey <edward.hervey@collabora.co.uk>
Date: Wed, 19 Sep 2012 11:01:18 +0200
Subject: [PATCH] video-sink: Remove usage of GL_TEXTURE_2D

GL header files are no longer exposed, and the Clutter Texture we use
are always 2D textures.
---
 clutter-gst/clutter-gst-video-sink.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/clutter-gst/clutter-gst-video-sink.c b/clutter-gst/clutter-gst-video-sink.c
index f8d3954..b6e4ad0 100644
--- a/clutter-gst/clutter-gst-video-sink.c
+++ b/clutter-gst/clutter-gst-video-sink.c
@@ -1090,10 +1090,6 @@ clutter_gst_hw_upload (ClutterGstVideoSink * sink, GstBuffer * buffer)
 
     tex = clutter_texture_get_cogl_texture (priv->texture);
     cogl_texture_get_gl_texture (tex, &gl_texture, &gl_target);
-    /* GL_TEXTURE_2D is always used in GLES2 */
-#ifndef COGL_HAS_GLES2
-    g_return_if_fail (gl_target == GL_TEXTURE_2D);
-#endif
 
     g_value_init (&value, G_TYPE_UINT);
     g_value_set_uint (&value, gl_texture);
-- 
1.8.0

